WalletWasabi.Fluent/App.axaml.cs

```diff
@@ -22,7 +22,7 @@ public class App : Application
 		/// Defines the <see cref="CanShutdownProvider"/> property.
 		/// </summary>/
 		public static readonly StyledProperty<ICanShutdownProvider?> CanShutdownProviderProperty =
-			AvaloniaProperty.Register<App, ICanShutdownProvider?>(nameof(CanShutdownProvider), null, defaultBindingMode: BindingMode.TwoWay);
+			AvaloniaProperty.Register<App, ICanShutdownProvider?>(nameof(CanShutdownProvider), new ApplicationViewModel(), defaultBindingMode: BindingMode.TwoWay);
```

Comment:
What you modified is the default value (if nothing is set), `null` was correct there. The real issue is that it doesn't get the value from the view although it is bound. It is because the `DataContext` is not set anywhere.

If you add this line
```
DataContext = new ApplicationViewModel();
```
to the `App()` constructor then it will get the value.

Reply:
Yes, you are right. Will do!